deploy/examples/object-user.yaml

```diff
@@ -11,6 +11,9 @@ metadata:
 spec:
   store: my-store
   displayName: "my display name"
```

Comment:
>  namespace: rook-ceph # namespace:cluster

We specify this in the metadata parameter,
`namespace: rook-ceph` 
make sense with this cahnge
so updating it to just

Reply:
As discussed, the `metadata.namespace` is where the object user CR is created, and `spec.clusterNamespace` is where the rook cluster is found.